pwnlib/tubes/remote.py

```diff
@@ -71,11 +71,10 @@ def __init__(self, host, port,
             self.settimeout(self.timeout)
             self.lhost, self.lport = self.sock.getsockname()[:2]
 
-            if ssl:
-                try:
-                    self.sock = _ssl.wrap_socket(self.sock, **ssl)
-                except TypeError:
-                    self.sock = _ssl.wrap_socket(self.sock)
+            if type(ssl) == dict:
```

Comment:
Should use `isinstance()` rather than `type()` since someone might build a class which inherits from `dict`